### DISCUSSION TITLE
nothing appeared on the webpage after i cloud deployed in huggingface

### DISCUSSION THREAD

**Discussion by yuanlianglaw** [danny-avila/LibreChat#1280] [Troubleshooting]
a blank webpage, but it does showing that the project is running. 
![image](https://github.com/danny-avila/LibreChat/assets/66453801/96831d3a-d3e1-411f-9b09-f03134642abb)
where am i wrong
help please

**Comment by fuegovic** [comment]
Hard to tell what went wrong, try to click beside the name to see the logs... The demo has been updated to support more features but might not be as easy as it used to to clone. I'll try to find the time to make a more basic variant which will make things easier for someone who wants to get started with it.

**Reply by fuegovic** [reply]
@yuanlianglaw Try duplicating this new template I made instead:
https://huggingface.co/spaces/LibreChat/template

It uses a more basic configuration and should be a better starting point than the one you used
